admin/test/system/workarea/admin/create_segments_system_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+
+module Workarea
+  module Admin
+    class CreateSegmentsSystemTest < Workarea::SystemTest
```

Comment:
This kinda needs to be a system test because we're testing the presence of an `<input type="hidden">` in the form. Anything else would, in my opinion, be mocking out things too much and could result in a regression without failing the test suite.

Reply:
I agree, but it also seems heavy handed to add an entire system test for one hidden field. I'm trying to be more sensitive to the performance costs of system tests, so I'd prefer to either take it out or use an `assert_select` in an integration test.

Reply:
I'm just gonna remove it, not sure if constantly having a test for this is worth rethinking everything.